lib/suite.js

```diff
@@ -123,9 +121,6 @@ Suite.prototype.timeout = function(ms) {
   if (!arguments.length) {
```

Comment:
`Suite.prototype.timeout` is different than `Runnable.prototype.timeout`, eg. there is no range check.
I guess that's out of scope of this PR.

Reply:
it should be clamped tho